### ISSUE TITLE
capacitor-document-scanner is not compatible with @capacitor/core 6.0.0

### ISSUE THREAD

**Issue by mantosh59** [WebsiteBeaver/capacitor-document-scanner#21]
capacitor-document-scanner is not compatible with @capacitor/core 6.0.0

**Comment by afilp** [comment]
agree, we also have this problem, maybe it is just a simple version bump in peer dependencies and it will work.
 
 I just did a PR for this:
 https://github.com/WebsiteBeaver/capacitor-document-scanner/pull/22

**Comment by mantosh59** [comment]
Please merge asap if everything is working fine after the upgradation.

**Comment by fatihtkale** [comment]
@mantosh59 Did you find a work around or some other way to do it?

**Comment by umarnisar1** [comment]
any update on this pr

**Comment by fatihtkale** [comment]
I think the devs are not mainting this anymore

**Comment by fatihtkale** [comment]
if you guys want a quick solution you can install from my git with fixed 

`npm install https://github.com/fatihtkale/docusign.git`

**Comment by camilocalvo** [comment]
For those who want to upgrade themselves, here is a guide as well: https://capacitorjs.com/docs/updating/plugins/6-0.
The changes needed are actually very small.
